astropy/wcs/tests/test_tab.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+
+import numpy as np
+
+from astropy import wcs
+
+from . helper import SimModelTAB
+
+
+def test_2d_spacial_tab_roundtrip():
```

Comment:
Can you use the `tab_wcs_2di` pytest fixture here and in the test below instead of importing `SimModelTAB`?  For consistency with your other tests?